solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/update.test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('update environment type configs', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+  const envTypeId = setup.getSettings().get('envTypeId');
+  const envTypeConfigId = setup.getSettings().get('envTypeConfigId');
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
```

Comment:
You might want to do this in the `beforeEach` so that no data leaks between tests

Reply:
I'd like to discuss this and create a ticket to address it if needed. I took as base cost center integration tests and we need to have consistency